site/src/pages/ChatPage/ChatLanding.tsx

```diff
@@ -0,0 +1,184 @@
+import { useTheme } from "@emotion/react";
+import SendIcon from "@mui/icons-material/Send";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Stack from "@mui/material/Stack";
+import TextField from "@mui/material/TextField";
+import { createChat } from "api/queries/chats";
+import type { Chat } from "api/typesGenerated";
+import { Margins } from "components/Margins/Margins";
+import { useAuthenticated } from "hooks";
+import { type FC, type FormEvent, useState } from "react";
+import { useMutation, useQueryClient } from "react-query";
+import { useNavigate } from "react-router-dom";
+import { LanguageModelSelector } from "./LanguageModelSelector";
+
+export interface ChatLandingLocationState {
+	chat: Chat;
+	message: string;
+}
+
+const ChatLanding: FC = () => {
+	const { user } = useAuthenticated();
+	const theme = useTheme();
+	const [input, setInput] = useState("");
+	const navigate = useNavigate();
+	const queryClient = useQueryClient();
+	const createChatMutation = useMutation(createChat(queryClient));
+
+	const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+		setInput(event.target.value);
+	};
+
+	// Placeholder submit handler
+	const handleFormSubmit = (e: FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+		if (!input.trim()) return;
```

Comment:
What is the reason on having this?

Reply:
This prevents submitting empty messages.

Reply:
You could just add a "required" prop in the text area or input.